### ISSUE TITLE
Corrections in osm

### ISSUE THREAD

**Issue by alxpapadaki** [FruitsofThought/ngambomap#1]
(This is an overarching issue. issue should be split up)

Regarding Architectural typologies:
-changes in the tag names:
1. Double Storey Swahili becomes "Modern Multi-storey Swahili"
2. Swahili 2000 or Swahili 2020 becomes "Post-modern Multi-storey Swahili"
3. Kariakoo becomes "Arcade building"
4. New tag: "Modern Indian Shopfront"

Regarding Architectural style:
-changes in the tag names:
1. First Swahili becomes "Classic Swahili"
2. Stone town becomes 2 different styles: "Omani" and "Indian"
3. "Omani Architecture" doesn't exist.
4. Modernist dosen't exist. It's only "Modernism"
5. Historicising is deleted and everything is becoming "Eclectic"

...and what is "C"? ...

Building Material and Roof Material keys don't exist in this map.

Regarding Building Structure:
-changes in the tag names:
1. The names should be "Coralstone AND lime", and  "Concrete frame AND cement blocks"
-notes:
1. Timber Frame and Metal Sheets is not a building structure
2. Wattle and daub still not visible

Regarding Roof Structure:
-changes in the tag names:
1. The name should be "Timber frame AND Metal Sheets"
-notes:
1. 2 tags not visible yet

Regarding Building Condition:
perfect. good job.

Regularly updated guide for the names here:
https://docs.google.com/document/d/1wypvZNLMSU8PRU0MdrQxqV2B4gb43oA1bMLaWL_71EE/edit?usp=sharing

**Comment by MariekeOosterom** [comment]
I'm sorry, I don't understand.. Can we just make these changes in josm? In a file from this repository, and then push it? Or do we need to programm the changes with the commands?

**Comment by batje** [comment]
Yes, these changes need to be made with Josm, then uploaded to OSM. Once that is done, the QGIS map needs to use the new values to render (nice) colors. So part of this big issue is fixing tags (the once with underscores) in OSM, and another part is then updating the QGIS files. I actually split off these issues for Urban Typologies ( eg. issue #2 & #3) to make this more managable.
